test/graphql_generator_test.dart

```diff
@@ -104,8 +104,10 @@ void main() {
     // ///Read the generated result from the file as String.
     var result = await resultFile.readAsString();
 
-    // ///Compare both the results.
-    expect(expectedResult, result);
+    DiffMatchPatch dmp = new DiffMatchPatch();
+    List<Diff> differences = dmp.diff(expectedResult, result);
+    print(
```

Comment:
@venkatd Yes, we can check if the response is equal to the local file. But since we are comparing the local file to a remote response the output will fail for a small change in the GraphQL response. I added only the diff to simplify the output shown.